### PR TITLE
[plugin.audio.kxmxpxtx.bandcamp] v0.5.0+matrix.1

### PR DESCRIPTION
**Pull request by molzy**
### Description
 - Added image quality settings and band fanart
 - Moved grouping by artist to submenu
 - Added icons to main menu
 - Add German and Dutch translations

Note that the original maintainer of this plugin is @VirusMater. I wanted to make some improvements, and am submitting these changes myself [per their request in this comment](https://github.com/Virusmater/plugin.audio.kxmxpxtx.bandcamp/pull/9#issuecomment-1290422577). I have updated the source link in `addon.xml` to my fork of the repository, but have not yet added my name or email contact as the provider in `addon.xml` as I'm not sure whether that would be appropriate at this stage.

The German translation is adapted from a [forum post by the user 'peat'](https://forum.kodi.tv/showthread.php?tid=354190&pid=2974663#pid2974663). The Dutch translation was created by me.

### Checklist:

- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.audio.kxmxpxtx.bandcamp/addon.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.audio.kxmxpxtx.bandcamp" name="Bandcamp" version="0.4.1+matrix.1" provider-name="kxmxpxtx">
+<addon id="plugin.audio.kxmxpxtx.bandcamp" name="Bandcamp" version="0.5.0+matrix.1" provider-name="kxmxpxtx">
```
Comment (basrieter) [inline plugin.audio.kxmxpxtx.bandcamp/addon.xml]: Perhaps update the Provider name? As you are more or less taking over?

**Review by basrieter — CHANGES_REQUESTED** [review]
Please check my minor change suggestion?

File: plugin.audio.kxmxpxtx.bandcamp/addon.xml (hunk 1 above)
Comment (molzy) [reply plugin.audio.kxmxpxtx.bandcamp/addon.xml]: @Virusmater stated that they are "not really interested anymore in this plugin" in the comment I linked earlier. So I think it's a good idea to take over as the provider for now. I can at least keep it running on future Kodi versions, as well as hopefully adding more features over time.

I'll add the changed provider to this PR.

**Comment by molzy** [comment]
I had left the below paragraph in an inline comment on your suggestion @basrieter, but thought I’d make it a full comment in case it was hidden away.

@Virusmater stated that they are "not really interested anymore in this plugin" in the comment I linked earlier. So I think it's a good idea to take over as the provider for now. I can at least keep it running on future Kodi versions, as well as hopefully adding more features over time.

I have added the changed provider to this PR. Let me know if there’s anything else I should do.

And thanks :)

**Review by enen92 — APPROVED** [review]

**Pull request merged**